native/src/lib.rs

```diff
@@ -46,10 +46,10 @@ declare_types! {
                 .check_argument::<JsString>(0)
                 ?.value();
 
-            // let mut wtr = io::BufWriter::new(File::create(filename).unwrap());
+            let wtr = io::BufWriter::new(File::create(filename).unwrap());
```

Comment:
this doesn't need to be mutable, because you never change the binding. buffers can be operated on without being mutable, just like `Vec`